### ISSUE TITLE
contentScript.js:23 Cross-Origin Read Blocking (CORB) blocked cross-origin response http://127.0.0.1:4200/ with MIME type application/json.

### ISSUE THREAD

**Issue by SvenMeyer** [zilpay/zil-pay#21]
Trying to access kaya local block chain from Zilpay.
```
$ node src/server.js -v -f test/account-fixtures.json

ZILLIQA KAYA RPC SERVER (ver: 0.2.9)
Server listening on 127.0.0.1:4200
Running from remote interpreter
...
```

```javascript
      const zilliqa = window.zilPay;
      const blockchain = zilliqa.blockchain;
      const result = await blockchain.getBlockChainInfo();
```

Chrome console:

contentScript.js:23 Cross-Origin Read Blocking (CORB) blocked cross-origin response http://127.0.0.1:4200/ with MIME type application/json. See https://www.chromestatus.com/feature/5629709824032768 for more details.

Chrome Network:
Request Payload
```
{"id":1,"jsonrpc":"2.0","method":"GetBlockchainInfo","params":[]}
```
![image](https://user-images.githubusercontent.com/25609/70705565-5c244680-1d28-11ea-8c4e-70f88203225a.png)

**Comment by SvenMeyer** [comment]
{"id":1 , ... , but kaya blockchain has id = 111 , maybe that is the problem ?

**Comment by SvenMeyer** [comment]
It is working on Testnet.

**Comment by lich666dead** [comment]
Thanks for your issue i will fixed it. I will just add new permissions to manifest.json

**Comment by lich666dead** [comment]
fixed by release [0.3.5](https://github.com/zilpay/zil-pay/tree/0.3.5)
